charts/gateway-helm/values.tmpl.yaml

```diff
@@ -1,9 +1,27 @@
+# The global settings for the Envoy Gateway Helm chart.
+# These values will be used if the values are not overridden in the other sections.
+global:
```

Comment:
Not sure a `global` umbrella makes sense here. 

If may be other `global` options in the value files, however, the EG value file is not that large.